pkg/driver/driver.go

```diff
@@ -193,6 +193,10 @@ func (d *Driver) Run() error {
 	csi.RegisterNodeServer(d.Srv, d.NodeServer)
 
 	klog.Infof("Listening for connections on address: %#v", listener.Addr())
+
+	// Start taint watcher when gRPC server is ready to accept connections
+	go node.StartNotReadyTaintWatcher(d.NodeServer.Clientset, node.TaintWatcherDuration)
```

Comment:
The `d.NodeServer.Clientset` might be `nil` (it's `nil` in tests for example) but we don't check if it's `nil` here, and it might cause a runtime panic if we just attempt to use it. We should probably put this into an if block.